sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -143,6 +148,24 @@ above.
 
 # Use SYCL toolchain
 
+## Using the SYCL toolchain on CUDA platforms
+
+The SYCL toolchain support on CUDA platforms is still on an experimental phase.
```

Comment:
still in an experimental phase

Reply:
Addressed in 7d027efeda7845a8d6df6f7fa109d4c73451ea61